terracumber_config/tf_files/SUSEManager-4.2-beta-build-validation.tf

```diff
@@ -755,6 +757,59 @@ module "ubuntu1604-minion" {
 
 }
 
+// TODO: When we enable debian in sumaform enable these
+/*
+module "debian9-minion" {
+  providers = {
+    libvirt = libvirt.florina
+  }
+  source             = "./modules/minion"
+  base_configuration = module.base_newsle_ubuntu.configuration
+  product_version    = "4.2-beta"
+  name               = "min-debian9"
+  image              = "debian9o"
+  provider_settings = {
+    mac                = "aa:b2:92:42:d1:bd"
```

Comment:
No. The MAC address should probably start with `aa:b2:92:42:00`, and its last byte should be identical to the last byte of its IP address. Check in gitlab's galaxy/infrastructure.